src/models/newJobHandler.ts

```diff
@@ -15,9 +15,15 @@ export class NewJobHandler implements IJobHandler {
     };
   }
 
-  public async handleJob(job: IJobResponse<NewRasterLayer, unknown>): Promise<void> {
-    const logCtx: LogContext = { ...this.logContext, function: this.handleJob.name };
-    this.logger.info({ msg: 'handling new job', metadata: { job }, logContext: logCtx });
+  public async handleJobInit(job: IJobResponse<NewRasterLayer, unknown>): Promise<void> {
+    const logCtx: LogContext = { ...this.logContext, function: this.handleJobInit.name };
+    this.logger.info({ msg: 'handling "new" job "init"', metadata: { job }, logContext: logCtx });
+    await Promise.reject('not implemented');
+  }
+
+  public async handleJobFinalize(job: IJobResponse<NewRasterLayer, unknown>): Promise<void> {
+    const logCtx: LogContext = { ...this.logContext, function: this.handleJobFinalize.name };
+    this.logger.info({ msg: 'handling "new" job "finalize"', metadata: { job }, logContext: logCtx });
```

Comment:
```suggestion
    this.logger.info({ msg: 'handling "new" job with "finalize" task', metadata: { job }, logContext: logCtx });
```

Reply:
the current log kinda confused me with job and task type, consider my suggest or rephrase it please to be more clearly

Reply:
changed